core-web/package.json

```diff
@@ -37,16 +37,13 @@
     "format": "nx format:write",
     "format:write": "nx format:write",
     "format:check": "nx format:check",
-    "update": "nx migrate latest",
-    "workspace-generator": "nx workspace-generator",
     "dep-graph": "nx dep-graph",
-    "help": "nx help",
     "pretty-quick": "pretty-quick",
-    "prepare": "is-ci || cd ../ && pwd && husky install core-web/.husky"
+    "prepare": "cd .. && node core-web/prepare.js"
```

Comment:
This change was introduce by Victor to fix the CICD, are you sure this won't break it again?

Reply:
I sent the change to him.